README.md

```diff
@@ -137,6 +137,24 @@ To use the Docker image, run:
 
 	docker run --rm -v $PWD:/mnt -w /mnt my:tag <shfmt arguments>
 
+### pre-commit
+
+It is possible to use shfmt with [pre-commit][pre-commit] and a `local`
+repo configuration like:
+
+```yaml
+  - repo: local
+    hooks:
+      - id: shfmt
+        name: shfmt
+        minimum_pre_commit_version: 2.4.0
+        language: golang
+        additional_dependencies: [mvdan.cc/sh/v3/cmd/shfmt@v3.1.1]
+        entry: shfmt
+        args: [-w, -s]
```

Comment:
the only flag that we should enable by default is `-w`, since we want to modify the files in place.

Reply:
Sure, `-s` is just a personal preference that I always use. But dropped it and force-pushed.